docs/components/modeler/bpmn/script-tasks/script-tasks.md

```diff
@@ -25,13 +25,13 @@ Jobs for script tasks are not processed by Zeebe itself. To process them, provid
 
 ## Defining a task
 
-A script task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition). It specifies
+A script task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition) the same way as a service task does. It specifies
 the type of job workers should subscribe to (e.g. `script`).
 
 Use [task headers](/components/modeler/bpmn/service-tasks/service-tasks.md#task-headers) to pass static parameters to the job
-worker (e.g. the script to evaluate).
+worker (e.g. the script to evaluate). The community extension [Zeebe Script Worker](https://github.com/camunda-community-hub/zeebe-script-worker) requires certain attributes to be set in the task headers.
```

Comment:
👍 I like this change

💭  Do we have somewhere documentation about task headers that we could link to here?

Reply:
Isn't documentation about task headers linked in line 31? Or are you thinking about something different here?

Reply:
Haha, yes you're absolutely right! I missed that one 😅

Reply:
Okay - anything else missing? 
(I am still in need for an approval here)